.github/workflows/e2e-manual.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       - name: Generate testnets
         working-directory: test/e2e
         # When changing -g, also change the matrix groups above
-        run: ./build/generator -g 4 -d networks/nightly/
+        run: ./build/generator -g 5 -d networks/nightly/
```

Comment:
Should we maybe also do this for v36?

Reply:
could do, though those tests aren't failing regularly because of timeouts, so I think it's fine for now.